CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.0 (2020-03-19)
+
+- Added the `maxFileSize` input to the avatar component. [#17](https://github.com/blackbaud/skyux-avatar/pull/17) (Thanks [blackbaud-conorwright](https://github.com/blackbaud/skyux-avatar/commits?author=blackbaud-conorwright)!)
```

Comment:
Is the second link correct? Don't we usually link to the user profile? This links to a commits page.